lang/cs/Org.Apache.REEF.IMRU.Examples/MapperCount/MapperCount.cs

```diff
@@ -66,8 +72,16 @@ public int Run(int numberofMappers)
                             GenericType<IntSumReduceFunction>.Class)
                         .Build())
                     .SetPartitionedDatasetConfiguration(
-                        RandomInputDataConfiguration.ConfigurationModule.Set(RandomInputDataConfiguration.NumberOfPartitions,
+                        RandomInputDataConfiguration.ConfigurationModule.Set(
+                            RandomInputDataConfiguration.NumberOfPartitions,
                             numberofMappers.ToString()).Build())
+                    .SetResultHandlerConfiguration(
+                        TangFactory.GetTang()
+                            .NewConfigurationBuilder(fileSystemConfig)
```

Comment:
As mentioned above, you don't need to as file system configuration here. Just make sure set when you create an injecter for REEFClient.